mne_bids/write.py

```diff
@@ -328,7 +328,45 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     return fname
 
 
-def _coordsystem_json(raw, unit, orient, manufacturer, fname,
+def _electrodes_tsv(raw, fname, overwrite=False, verbose=True):
+    """Create an electrodes.tsv file and save it.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The data as MNE-Python Raw object.
+    fname : str
+        Filename to save the electrodes.tsv to.
+    overwrite : bool
+        Defaults to False.
+        Whether to overwrite the existing data in the file.
+        If there is already data for the given `fname` and overwrite is False,
+        an error will be raised.
+    verbose : bool
+        Set verbose output to true or false.
+
+    """
+    x, y, z, names = list(), list(), list(), list()
+    for ch in raw.info['chs']:
+        if _check_ch_locs([ch]):
+            x.append(ch['loc'][0])
+            y.append(ch['loc'][1])
+            z.append(ch['loc'][2])
+        else:
+            x.append('n/a')
+            y.append('n/a')
+            z.append('n/a')
+        names.append(ch['ch_name'])
+    data = OrderedDict([('name', names),
+                        ('x', x),
+                        ('y', y),
+                        ('z', z),
+                        ])
+    _write_tsv(fname, data, overwrite=overwrite, verbose=verbose)
+    return fname
+
+
+def _coordsystem_json(raw, unit, orient, coordsystem_name, fname, kind,
```

Comment:
do you have a default `coordsystem_name`? is this a single coord_frame for all poits or they all can have different reference frame? 

in MNE we set `'unknown'`. If I'm not mistaken here I would be restrictive here and only allow for reference names that are validated. 

i.e:
```py
VALIDATED_VENDOR_COORD_FNAME = {'xx', 'yy'}
_check_output(coordsystem_name, VALIDATED...)
```

also, maybe we should discuss parameter names. We have been rethinking some of them in mne-python#6639. I would love to hear your inputs.

Reply:
see result of talk here: https://github.com/mne-tools/mne-bids/issues/264